winterfell/webapp/gulpfile.js

```diff
@@ -129,6 +138,6 @@ gulp.task('watch', ['build', 'initBrowserSync'], function () {
   gulp.watch('src/**/*.jade', ['jade']);
```

Comment:
well all it does right now is spam a bunch of stuff out to the console, which would be annoying during rapid iteration

i think it should just be in the main build, and we can have it fail the build task if there are lint errors, which will force us to keep our code clean

i didnt turn that feature on yet, because we still need to clean up all the code